### ISSUE TITLE
[Fellowship Mini Site] link Pulse profiles to names (instead of Twitter Profiles)

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/foundation.mozilla.org#1136]
### Issue
we currently have a mix of Pulse and Twitter profiles links in our [Fellowship Home Page:](https://foundation-mofostaging--pr-996.herokuapp.com/fellowships/)
<img width="676" alt="image" src="https://user-images.githubusercontent.com/22157921/36921405-1fcada24-1e19-11e8-865c-d7c46ba443a3.png">

Ideally, once we have all Pulse Profiles up, we would link all names to Pulse pages.

**Comment by taisdesouzalessa** [comment]
Dependency: having the Fellowship mini site set on staging so I can do the change in my local machine and push to GH. @mmmavis could you please let me know once the transition to staging is done?

**Comment by mmmavis** [comment]
@taisdesouzalessa these links are hardcoded in the copy so you can create your branch based off of the `fellowships` branch and make copy changes there. I can help you with this tomorrow when we are both in the office.

**Comment by taisdesouzalessa** [comment]
Yay! thanks @mmmavis. I will definitely need your help. Unblocking the issue then

**Comment by taisdesouzalessa** [comment]
Updated and created a PR. Some notes:

1. Steve Song's profile is missing from Pulse (using his [Twitter profile](https://twitter.com/stevesong) for now)
2. [Suchana Seth's profile](https://www.mozillapulse.org/profile/330) is pretty incomplete by now. It would be nice to have more info and a picture there: 
3. [Sarah Kiden's profile](https://www.mozillapulse.org/profile/312) is also very empty

@annmarie123 if possible could you help us to fill in those profiles in Pulse (or ask fellows to do so?) If not, it's not a problem, we could use the Twitter handle in those cases or leave as is. For reference, this is the mockup of the page and the names in blue take users to Pulse profiles when they exist:
https://invis.io/ZJ9YTAGEA#/281947967_v2-FellowshipsUI_-_Overview

**Comment by annmarie123** [comment]
@taisdesouzalessa I have updated all those profiles and am currently working on double checking all current fellow profiles.
